<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libpcap</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="source_packet">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>561</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="netinfo_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>421</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableView" name="packet_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>551</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
   </widget>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
